temporalcli/commandsmd/commands.md

```diff
@@ -1025,6 +1025,23 @@ Use the options listed below to change the behavior of this command.
 * `--reason` (string) - Reason for termination. Defaults to message with the current user's name.
 * `--yes`, `-y` (bool) - Confirm prompt to perform batch. Only allowed if query is present.
 
+### temporal workflow trace: Terminate Workflow Execution by ID or List Filter.
+
+The `temporal workflow trace` command display the progress of a [Workflow Execution](/concepts/what-is-a-workflow-execution) and its child workflows with a trace.
+This view provides a great way to understand the flow of a workflow.
+
+Use the options listed below to change the behavior of this command.
+
+#### Options
+
+* `--fold` (string) - Statuses for which Child Workflows will be folded in (this will reduce the number of information fetched and displayed). Case-insensitive and ignored if no-fold supplied. Available values: running, completed, failed, canceled, terminated, timedout, continueasnew. Default: completed,canceled,terminated.
```

Comment:
If we supported it, would suggest `string-enum[]` as the type, but in the meantime, might as well just have `string[]` as the type. This allows them to specify `--fold` multiple times instead of using commas. However, default not supported for `string[]`. You can either add support in `commandsmd/code.go` or just have something like "Default is completed, canceled, terminated" which won't trigger the parser and let you set the default at runtime.

Reply:
I'm happy to do this is a later iteration so we can move this (already pretty big thing) forward. Does this work for you?

Reply:
Sure, but it will be a backwards-incompatible alteration at that time unfortunately.

Reply:
Can you pass multiple statuses to a single instance of `--fold`? The help text says "Statuses", which makes me think this is possible, but we should make it explicit in the help text that commas are needed.

Reply:
Could I just not use the flag for now and we can figure out how to work this out in a future PR? I think it's a non-trivial decision that we would benefit from putting some good thinking behind (and not delay this PR).

Reply:
Will defer to @josh-berry. I do think however it is fairly trivial to just change this to `string[]` and wait until you see an empty slice at runtime to set the default 3 values.

Reply:
Not having the flag and adding it later is fine by me; so is accepting `[]string`. The only thing I'm not thrilled about is making a breaking change (i.e. commas now and `[]string` later).

Reply:
:+1: @sebneira - can you make this type `string[]` and set the defaults in code on empty slice when no-fold is not set? You can still write "Defaults are completed, canceled, and terminated." and it won't be read as a special option like `Default:` is.

Reply:
Thanks for the suggestion. I updated the PR to include the `string[]` flag for fold which defaults to the completed, canceled and terminated if the flag is empty.

Also, as I was moving the options struct around it kinda dawned on me that the pkg made more sense as `tracer` rather than `trace`. Let me know if this doesn't make sense.